packages/dev-server/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@expo/babel-preset-cli/tsconfig.base",
+  "include": ["src/**/*.ts"],
```

Comment:
Be sure to exclude tests:
```json
{
  "extends": "@expo/babel-preset-cli/tsconfig.base",
  "compilerOptions": {
    "outDir": "build",
    "rootDir": "src"
  },
  "include": ["src/**/*.ts"],
  "exclude": ["**/__mocks__/*", "**/__tests__/*"]
}
```

Reply:
I want `__tests__` to be type checked however, as far as I know Jest compiles them with Babel which doesn't do type checking? Maybe we can only ignore them in `.npmignore` or `files` when publishing to npm?